sdkconfig.defaults

```diff
@@ -24,3 +25,33 @@ CONFIG_ESP_SECURE_CERT_SUPPORT_LEGACY_FORMATS=y
 CONFIG_GRI_SUB_PUB_UNSUB_DEMO_MAX_COMMAND_SEND_BLOCK_TIME_MS=10000
 CONFIG_GRI_SUB_PUB_UNSUB_DEMO_DELAY_BETWEEN_SUB_PUB_UNSUB_LOOPS_MS=10000
 CONFIG_GRI_MQTT_AGENT_CONNACK_RECV_TIMEOUT_MS=10000
+
+# Set optimization for size
+CONFIG_COMPILER_OPTIMIZATION_SIZE=y
+
+# Set default FreeRTOS Stack sizes
+CONFIG_FREERTOS_TIMER_TASK_STACK_DEPTH=1536
+CONFIG_FREERTOS_IDLE_TASK_STACKSIZE=1024
+
```

Comment:
These too

Reply:
I got them by running `idf.py menuconfig` and selecting the minimal supported task stack size for these features

I ran the demos for a bit and didn't see the stack overflow check get raised, so don't believe the sizes are needed. The previous defaults were `1536` for the idle task and `2048` for the timer task.